libs/output-mapping/tests/Writer/Table/MetadataSetterTest.php

```diff
@@ -37,58 +42,224 @@ public function testSetTableMetadata(
         );
 
         self::assertCount($expectedCountMetadata, $loadTask->getMetadata());
+        self::assertEquals($expectedMetadata, $loadTask->getMetadata());
     }
 
     public function provideSetTableMetadata(): Generator
     {
         $mappingDestination = new MappingDestination('in.c-main.table');
+        $tableLastUpdateMetadata = new TableMetadata(
+            'in.c-main.table',
+            'system',
+            [
+                [
+                    'key' => 'KBC.lastUpdatedBy.component.id',
+                    'value' => 'keboola.test',
+                ],
+            ],
+        );
+        $tableCreatedByMetadata = new TableMetadata(
+            'in.c-main.table',
+            'system',
+            [
+                [
+                    'key' => 'KBC.createdBy.component.id',
+                    'value' => 'keboola.test',
+                ],
+            ],
+        );
+        $tableMetadata = new TableMetadata(
+            'in.c-main.table',
+            'keboola.test',
+            [
+                [
+                    'key' => 'key1',
+                    'value' => 'val1',
+                ],
+                [
+                    'key' => 'key2',
+                    'value' => 'val2',
+                ],
+            ],
+        );
+        $columnMetadata = new ColumnMetadata(
+            'in.c-main.table',
+            'keboola.test',
+            [
+                [
+                    'key' => 'col-key1',
+                    'value' => 'col-val1',
+                ],
+                [
+                    'key' => 'col-key2',
+                    'value' => 'col-val2',
+                ],
+            ],
+        );
+        $schemaColumnMetadata = new SchemaColumnMetadata(
+            'in.c-main.table',
+            'keboola.test',
+            [
+                new MappingFromConfigurationSchemaColumn([
+                    'mapping' => [
+                        'description' => 'column desc',
+                        'metadata' => [
+                            'key1' => 'val1',
+                            'key2' => 'val2',
+                        ],
+                    ],
+                ]),
+            ],
+        );
 
         $mockMappingFromProcessedConfiguration = $this->createMock(MappingFromProcessedConfiguration::class);
```

Comment:
Ted to neni blocker, ale od phpunit10 to bude deprecated a pak to odeberou uplne
> Using a non-static method as a data provider is now deprecated